web/marketing/css/style.css

```diff
@@ -6,26 +6,117 @@ html{
 body{
     height:100%;
     width:100%;
-    background-color: #DDDDDD;
+    background-color:#E5E5E5;
 }
 
-.home-link:hover,.home-link p:hover{
+.home-link:hover,
+.home-link p:hover{
     text-decoration: none;
 }
 
+.btn{
+    height:48px;
+}
+
+hr{
+    margin-top:-5px;
+    margin-left: -15px;
+    margin-bottom: 0px;
+    width: 1102px !important;
```

Comment:
Why do we need all the `!important` here?

Reply:
No. I removed it.